code_fellows/deque.py

```diff
@@ -0,0 +1,74 @@
+# https://codefellows.github.io/sea-python-401d6/assignments/deque.html
+
+
+class Deque:
+    def __init__(self):
```

Comment:
why did you get rid of values?

Reply:
Because I couldn't figure out what it was doing. But now that you ask, I'll have to look back at the assignment. I was thinking it was invoked empty then items are added. But maybe it needs to be able to handle being instantiated with items like test_deq = Deque("one", 2, "z")?

I think the previous assignment invoked empty and I might have just carried that on through to this one. 

I actually just checked and it doesn't say... So, I'm not sure now. I'm assuming you made yours to handle iterating things on instantiation?

Reply:
In this case the assignment doesn't say that it should start with values, so removing it here is a-okay.